src/pages/question/subComponents/ButtonGroupPanel.jsx

```diff
@@ -11,8 +11,8 @@ import { FaPlusSquare, FaPlus } from 'react-icons/fa';
  * @param {function} toggleJsonEditor function to close the advanced json editor
  */
 export default function ButtonGroupPanel({ panelStore, toggleJsonEditor }) {
-  const numNodes = Object.entries(panelStore.query_graph.nodes)
-    .filter(([id, node]) => !node.deleted).length;
+  const numNodes = Object.keys(panelStore.query_graph.nodes)
+    .filter((id) => !panelStore.query_graph.nodes[id].deleted).length;
```

Comment:
Instead of keys, this can be Object.values(), which will give you each node object. The way you've done it works, but isn't the most optimal.